### PR TITLE
Gh 288 deduplicate results

### PR DESCRIPTION
**Pull request by p013570**

### CODE REVIEW

**Comment by GCHQ-11** [comment]
Could de-duplication be done as a separate step in an Operation Chain? It could be implemented in the general operations rather than being Accumulo specific.

**Comment by p013570** [comment]
I'll make the suggested chain and then reopen the pull request.

**Comment by p013570** [comment]
I've now created a Deduplicate operation. There is now a deduplicate flag on GetOperation that will automatically add a deduplicate operation into your operation chain for you.

File: gaffer-core/graph/src/main/java/gaffer/graph/Graph.java [hunk 1]
```diff
@@ -110,20 +110,25 @@ private Graph(final Store store, final View view, final List<GraphHook> graphHoo
...
+        final OperationChain<OUTPUT> operationChainClone = operationChain.clone();
+
+        // Update the view
+        for (Operation operation : operationChainClone.getOperations()) {
             if (null == operation.getView()) {
                 operation.setView(view);
             }
         }
 
         for (GraphHook graphHook : graphHooks) {
```
Comment (a09631) [inline gaffer-core/graph/src/main/java/gaffer/graph/Graph.java]: Mark graphHook final

**Pull request merged**